consensus/spos/worker.go

```diff
@@ -384,8 +372,13 @@ func (wrk *Worker) ProcessReceivedMessage(message p2p.MessageP2P, fromConnectedP
 
 	go wrk.updateNetworkShardingVals(message, cnsDta)
 
+	isMessageWithBlockBody := wrk.consensusService.IsMessageWithBlockBody(msgType)
 	isMessageWithBlockHeader := wrk.consensusService.IsMessageWithBlockHeader(msgType)
 	isMessageWithBlockBodyAndHeader := wrk.consensusService.IsMessageWithBlockBodyAndHeader(msgType)
+	if isMessageWithBlockBody || isMessageWithBlockHeader {
```

Comment:
if isMessageWithBlockBody || **isMessageWithBlockBodyAndHeader** ?

Reply:
yes, that was the intention
good find